lisa-utils/src/main/scala/lisa/utils/tactics/BasicStepTactic.scala

```diff
@@ -14,6 +14,13 @@ import lisa.utils.tactics.ProofTacticLib.{_, given}
 
 object BasicStepTactic {
 
+  def unwrapTactic(using proof: Library#Proof)(using tactic: ProofTactic)(judgement: proof.ProofTacticJudgement)(message: String): proof.ProofTacticJudgement = {
```

Comment:
@SimonGuilloud I'm not sure if this should stay in `BasicStepTactic` or be moved to say, `ProofHelpers`. Any preference?

Reply:
Good question. Maybe in ProofHelpers, as it would also be usefull for future tactics.

Reply:
I am not entirely sure what would be the best place for it atm. I will move it while writing tests for some deduced tactics soon!